python/dgl/transforms/functional.py

```diff
@@ -1764,13 +1765,23 @@ def remove_nodes(g, nids, ntype=None, store_ids=False):
     g.remove_nodes(nids, ntype=ntype, store_ids=store_ids)
     return g
 
-def add_self_loop(g, etype=None):
+def add_self_loop(g, edge_feat_names=None, fill_data=1., etype=None):
     r"""Add self-loops for each node in the graph and return a new graph.
 
     Parameters
     ----------
     g : DGLGraph
         The graph.
+    edge_feat_names : list[str], optional
+        The names of the self-loops features to add data. If None, all added self-loops features are
```

Comment:
"The names of the self-loops features to add data" -> "The names of the self-loop features to apply ``fill_data``."